README.md

```diff
@@ -17,3 +17,329 @@ Implement a prototype job worker service that provides an API to run arbitrary L
 
 * Client should be able to query result of the job execution and fetch the logs
 
+## Design Document
+
+### Limitations And Scope
+
+#### Data Management
+Though the use of a database to store persistant data would be ideal, I will be instead storing the outputs and error outputs into logs stored on 
+the file storage of the linux worker. The logs will be generated at start time with the foldername \<pid\>_\<startingTimestamp\>
```

Comment:
thats a good point,
another idea i have floating around my head is generate a uuid with https://github.com/google/uuid and create a log folder:
<uuid>-<startingtimestamp> and if a process a created a log in the folder would be created called PID-<pid>-stdout.log and PID-<pid>-stderr.log and if it finishes it will generate a end log called END-<endTimeStamp>.log, the client will immediately get a response back from the start rpc with the pid and the uuid. When the client wants to query the status of a job, it should use the pid and the uuid, that way when the job is already completed, the user can still query the logs based on the uuid